compiler/testData/codegen/bytecodeListing/specialBridges/redundantStubForSize.kt

```diff
@@ -1,4 +1,3 @@
-// IGNORE_BACKEND: JVM_IR
```

Comment:
Would it be better to leave this one as not passing if there are still outstanding issues? If any of the remaining differences are not as expected, I would vote that we leave this one as not passing? It looks like the previous test covers the case that is fixed with this PR, so we have a regression test?

Reply:
Sure, good call.